### DISCUSSION TITLE
Stand up Friday May 19th

### DISCUSSION THREAD

**Discussion by CristianCazares** [Bugs-io/TC3001B#80] [Main Stand Ups]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
- 

**What are your priorities for the next 24 hours?**:
- 

**Blockers**:
- 

**Shoutouts**:
- @username for x
```

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
- Worked on the grid
- Implemented a way of saving the layout into LocalStorage (not sure about actually using it)

**What are your priorities for the next 24 hours?**:
- Make the layout more "generic".

**Blockers**:
- None

**Shoutouts**:
- @everyone bc we ONLY HAVE TRHEE WEEKS LEFT! (+ this weekend)

**Comment by Jacobgtz024** [comment]
**What did you achieve in the last 24 hours?**:
- Created endpoint with plotting data for the first graph
- Organized code into modules and files

**What are your priorities for the next 24 hours?**:
- Optimize the endpoint code
- PFKR
- Start working on the other graphs

**Blockers**:
- None

**Shoutouts**:
- @todos

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
- Worked on Luis Gerardo's assignment

**What are your priorities for the next 24 hours?**:
- Reach out to @Jacobgtz024 to organize the development of the graphs

**Blockers**:
- None

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
-  Worked on architecture document

**What are your priorities for the next 24 hours?**:
- Help @CarlosCRG19 with testing suite for backend

**Blockers**:
- Lack of will

**Shoutouts**:
- @Chivas por jugar bien el primer tiempo
